server/container/guice/guice-common/src/test/java/org/apache/james/utils/RetryExecutorUtilTest.java

```diff
@@ -39,7 +39,7 @@
 
 public class RetryExecutorUtilTest {
     private static final int MAX_RETRIES = 3;
-    private static final int MIN_DELAY = 3000;
+    private static final int MIN_DELAY = 00;
```

Comment:
you don't increase max retries?

Reply:
No, Iit's the class under test... And waiting 3 second (with exponential back-off) is such a waste...